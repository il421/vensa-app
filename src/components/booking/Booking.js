import React from 'react';
import Header from './Header';
import DoctorsList from './DoctorsList';

const Booking = () => (
  <>
    <Header />
    <DoctorsList />
  </>
);

export default Booking;
