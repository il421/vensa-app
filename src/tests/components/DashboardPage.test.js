import React from 'react';
import { shallow } from 'enzyme';
import Booking from '../../components/Booking';

test('should render Booking', () => {
  const wrapper = shallow(<Booking />);
  expect(wrapper).toMatchSnapshot();
});
