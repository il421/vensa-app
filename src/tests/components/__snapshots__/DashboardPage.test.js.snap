// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Booking 1`] = `
<div>
  Booking
</div>
`;
